### PR TITLE
docs: how to install into a spring boot project

### PR DESCRIPTION
**Pull request by netmikey**
To be merged once the modules are published to maven central.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -17,23 +17,73 @@ Embedded Keycloak Server | Keycloak
...
+
+To add Keycloak to a Spring Boot project, add a dependency to the Spring Boot starter and make sure to use this project's BOM/parent so that you're getting all the right dependency versions:
+
+In Maven:
+``` xml
+<project ...>
+  <parent>
+      <groupId>com.github.thomasdarimont.keycloak</groupId>
+      <artifactId>embedded-keycloak-server-spring-boot-parent</artifactId>
+      <version>2.1.0</version>
```
Comment (thomasdarimont) [inline readme.md]: Just released 2.2.0 :) Please update

**Review by thomasdarimont — CHANGES_REQUESTED** [review]
This looks great, thanks! I'll merge it once you have updated the versions.

File: readme.md (hunk 1 above)
Comment (netmikey) [reply readme.md]: Done :)

**Review by thomasdarimont — APPROVED** [review]
LGTM

**Comment by thomasdarimont** [comment]
Thanks!

**Pull request merged**